style.css

```diff
@@ -0,0 +1,103 @@
+/* Box Model Hack */
+* {
+  box-sizing: border-box;
+  font-family: Poppins, sans-serif;
+}
+
+/* Clear fix hack */
+.clearfix::after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+/******************************************
+  /* BASE STYLES
+  / *******************************************/
+body {
+  background-color: #e5e5e5;
+}
+
+@media only screen and (max-width: 768px) and (min-width: 400px) {
+  .after-body {
+    width: 375px;
+  }
+}
+
+.after-body {
+  padding: 10px 20px;
+  margin: 0 auto;
+  background: white;
+  border-bottom-left-radius: calc(10rem / 3);
+  overflow: auto;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 0;
+}
+
+/* coloring n font sizes */
+
+a {
+  text-decoration: none;
+}
+
+nav > a {
+  font-size: 18px;
+  font-weight: 700;
+  color: #6070ff;
+}
+
+#first-page {
+  background-image: url(images/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: right;
+  height: 90vh;
+}
```

Comment:
- [ ] The previous reviewer asked that you used flexbox to lay out this section. I understand you used flexbox for the `ul` tag under the sections for the ions, but you're meant to render the whole section with flexbox of which the `first-page` happens to be the `id` that holds that 👍